admin/assets/stylesheets/elements/_sorting.scss

```diff
@@ -0,0 +1,13 @@
+.sort {
+  display: inline-block;
+  width: 0;
+  height: 0;
+  border-style: solid;
+  border-width: 12px 6px 0 6px;
```

Comment:
How does this 12px border look on mobile?  Should we use `per()` to convert automatically to em?